repository_service_tuf/cli/artifact/__init__.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: MIT
+
+from typing import Optional
+
+from click import Context
+
+from repository_service_tuf.cli import click, rstuf
+
+
+@rstuf.group()
+@click.option(
+    "--rstuf-server",
+    help=(
+        "RSTUF API Server address. Not required if the `--auth` option is "
+        "passed."
+    ),
+    required=False,
+    default=None,
+)
+@click.option(
+    "-t",
+    "--token",
+    help=(
+        "RSTUF API authentication token. If the `--auth` option is provided"
+        "this token is not used for authentication."
+    ),
+    required=False,
+    default=None,
+)
```

Comment:
I'm down to doing that, but if we do this now it might confuse people that for commands that need `--auth`, `--token` can be interchangeably used instead for authentication.
I think we should do that when we actually support the use of it for other commands. What do you think?

Reply:
The `rstuf metadata [update|sign]` are already use cases. 

If the user has the admin he can use `--auth`. But if the user got from the admin a token to write metadata, the user must use `--token/-t`.

Reply:
I think we can add it in a separate PR. It's easy to review and merge before our features.
Using the option `—-token/-it is also valid for `rstuf admin ceremony` if admin gives you a scoped token for the bootstrap ceremony
WDYT?

Reply:
> If the user has the admin he can use --auth. But if the user got from the admin a token to write metadata, the user must use --token/-t.

Yep, that's what I had in mind as well. My point was that the other commands should first be adjusted to support the -`--token` option if we were to move it to `rstuf` command level.
With [this PR](https://github.com/repository-service-tuf/repository-service-tuf-cli/pull/348) the commands should be adjusted. It needs to be tested end-to-end with the `--token` option first though.